get-poetry.py

```diff
@@ -925,6 +926,13 @@ def main():
     parser.add_argument(
         "-f", "--force", dest="force", action="store_true", default=False
     )
+    parser.add_argument(
+        "-P",
```

Comment:
```suggestion
```
I think the short option might be more more confusing here, lets keep just the long option. Would also be great to add a help text here.